src/utils/propertyHelper.js

```diff
@@ -0,0 +1,28 @@
+export function getPropertyOrDefault(
```

Comment:
Maybe it's too late for me but, I don't understand the use case for this helper function and I wasn't able to find where did you use it. I don't have any concerns about it but, just would like to understand the use case for it.

Reply:
getPropertyOrDefault is used in localizationHelper, this is partly code we have been using in MSF and it does what we need to accomplished for localisations as we expect and as I told you yesterday. I will need getLocalization in the OMS and it will be useful throughout all our applications.

Reply:
It is just a helper function that allows us to get a property from an object with some use cases we had in MSF, eg, we needed to ignore case or we wanted to look for a property starting with some characters. Maybe it does sound silly for you but it has been useful in MSF in some use cases.